apiserver/admin.go

```diff
@@ -65,6 +65,12 @@ func (a *admin) login(req params.LoginRequest, loginVersion int) (params.LoginRe
 
 	if a.root.state.IsCAAS() {
 		// XXX don't bother with actual login checks here. Just fake it for the prototype for now.
+		u, err := a.srv.state.User(names.NewUserTag("admin"))
```

Comment:
Hrmmm, I would have thought this would work with user and API logins, since it is loading the user entity from the controller state. Regardless, what you're suggesting makes more sense so I'll rework it.